ironfist/src/cpp/combat/combat.cpp

```diff
@@ -596,14 +596,13 @@ void army::SpecialAttack() {
 	}
 }
 
-void army::LoadResources()
-{
-	if (!gbNoShowCombat)
-	{
-		int formFileID = gpResourceManager->MakeId(cArmyFrameFileNames[this->creatureIdx], 1);
+void army::LoadResources() {
```

Comment:
On the whole, this is good work. I might suggest changing the indices into combatSounds into named constants.